### ISSUE TITLE
Translations interface displays an empty languages list if collection name is blog_category

### ISSUE THREAD

**Issue by d1rOn** [directus/directus#7483]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

If you create a collection with the name exactly blog_category, add a field of type translations to it, then the translations interface does not display the list of languages on the pages for creating and editing collection data.
I could not find other combinations of the collection name in which this situation is repeated.

Languages collection:
![image](https://user-images.githubusercontent.com/26179694/129864001-836e9a11-d8ed-4e44-b2bb-d0fae8bc905b.png)

Blog Category collection:
![image](https://user-images.githubusercontent.com/26179694/129863878-d91bc89c-23d7-4612-8927-aff65c6baf48.png)

Collection with another name:
![image](https://user-images.githubusercontent.com/26179694/129871596-33674d3f-b663-4125-9b47-12d0c2745078.png)

### To Reproduce

1. Create colleaction with name blog_category.
2. Add field with translations interface.
3. Go to /admin/collections/blog_category/+.
4. See that the list of languages is not displayed in the interface.

### What version of Directus are you using?

9.0.0-rc90

### What version of Node.js are you using?

14.17.2

### What database are you using?

MySQL 8.0.25

### What browser are you using?

Chrome

### What operating system are you using?

Debian

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
I don't think this is related to the name of the collection necessarily. The translations interface doesn't do anything based on the collection name, and I wasn't able to reproduce it with that exact information. 

Would you be able to share a database dump where this is happening? I'm assuming a relationship might be missing for some reason, which causes the translations interface to fail

**Comment by rijkvanzanten** [comment]
Thank you for taking the time to submit this! Unfortunately, our team was not been able to reproduce it based on the information provided. Therefore, we will close this ticket for now, but will happily re-open it if new details are provided that allow us to reliably replicate the problem.

As a reminder:
* Provide as much detail as possible in the issue description
* Video screen captures and clear steps are very helpful
* A full SQL dump is the fastest way for us to test/resolve (can be sent privately)
* Make sure you are on the latest version of Directus
* Did you try turning it off and on again?

---

_This is an automated response._

**Comment by d1rOn** [comment]
MySQL dump: [directus-dump.zip](https://github.com/directus/directus/files/7019824/directus-dump.zip)

I was tested this dump on Windows, but was not been able to reproduce it too.

My Windows config:
- Node v14.17.2
- MySQL v8.0.19

**Comment by rijkvanzanten** [comment]
> I was tested this dump on Windows, but was not been able to reproduce it too.

Yeah, all looks good on my end on the latest version (rc.91, about to be released) too:

<img width="775" alt="CleanShot 2021-08-20 at 10 30 50@2x" src="https://user-images.githubusercontent.com/9141017/130249093-011e97c2-8623-46df-862d-aab2743b5a97.png">

**Comment by d1rOn** [comment]
I finally found the source of issue.
The root of the problem was that when the package knex-schema-inspector retrieves schema, my MySQL 8.0.25 returns not distinct results.
It cause /relations endpoint to return not distinct results too. And finally the translations interface tries to extract collection items list instead of languages list. The translations interface renders as an empty list when the collection is empty and as the list of empty boxes with length equals to collection items count when the collection is not empty.

I created a [pull request](https://github.com/knex/knex-schema-inspector/pull/67) to the knex-schema-inspector repository to fix this.
